### PR TITLE
Documentation update to explain FusedSDPA support for Llava fp8 inference

### PR DESCRIPTION
**Pull request by tthakkal**
# What does this PR do?
Updated readme to explain how to use FusedSDPA with fp8 inference for Llava models with 1.17

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: examples/image-to-text/README.md [hunk 1]
```diff
@@ -149,12 +149,33 @@ python3 run_pipeline.py \
...
     --bf16 \
     --use_flash_attention
-```
\ No newline at end of file
+```
+
+
+Use the following commands to run Llava-v1.6-mistral-7b FP8 inference with FusedSDPA
+
+Here is an example to measure the tensor quantization statistics on Llava-v1.6-mistral-7b:
```
Comment (yafshar) [inline examples/image-to-text/README.md:165]:
```suggestion
Here is an example of measuring the tensor quantization statistics on Llava-v1.6-mistral-7b:
```

File: examples/image-to-text/README.md [hunk 2]
```diff
@@ -149,12 +149,33 @@ python3 run_pipeline.py \
...
+```bash
+QUANT_CONFIG=./quantization_config/maxabs_measure.json python run_pipeline.py \
+--model_name_or_path llava-hf/llava-v1.6-mistral-7b-hf \
+--image_path "https://llava-vl.github.io/static/images/view.jpg" \
+--use_hpu_graphs \
+--bf16 --use_flash_attention
+```
+
+Here is an example to quantize the model based on previous measurements for Llava-v1.6-mistral-7b:
```
Comment (yafshar) [inline examples/image-to-text/README.md:174]:
```suggestion
Here is an example of quantizing the model based on previous measurements for Llava-v1.6-mistral-7b:
```

**Review by yafshar — APPROVED** [review]
LGTM!

@regisss could you please look at this

**Comment by regisss** [comment]
Closing as it was integrated into #1163.

**Pull request closed without merging**